### PR TITLE
 [14.0][MIG] report_csv: Migration to 14.0

### PR DESCRIPTION
**Pull request by tupaq**

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

image

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Comment by LoisRForgeFlow** [comment]
@tupaq there are some issues with pre-commit, could you rebase your branch and run it again? https://travis-ci.com/github/OCA/reporting-engine/jobs/421650683

**Comment by tupaq** [comment]
> @tupaq there are some issues with pre-commit, could you rebase your branch and run it again? https://travis-ci.com/github/OCA/reporting-engine/jobs/421650683

Rebased

**Comment by LoisRForgeFlow** [comment]
Something must have been not clean because you have a merge commit in your history, could you please try again? to clarify, when I say rebase I mean to do a git rebase operation in this pr's branch on top of latest oca/14.0.

File: report_csv/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Base report csv",
+    "summary": "Base module to create csv report",
+    "author": "Creu Blanca, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/reporting-engine",
+    "category": "Reporting",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "external_dependencies": {"python": ["python-csv"]},
```
Comment (davejames) [inline report_csv/__manifest__.py:11]: do we need to depend on python-csv? It seems that only csv is being used in the code which is what the dependency was in v13.

**Review by davejames — CHANGES_REQUESTED** [review]

**Pull request closed without merging**